src/org/labkey/test/components/glassLibrary/components/InsertEntitiesPanel.java

```diff
@@ -0,0 +1,193 @@
+package org.labkey.test.components.glassLibrary.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.glassLibrary.grids.EditableGrid;
+import org.labkey.test.components.react.dialogs.BulkCreateSamplesDialog;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Map;
+
+// This is the same component (collection of atomic elements) used in insertAssay and createSamples.
+
+public class InsertEntitiesPanel extends WebDriverComponent<InsertEntitiesPanel.ElementCache>
```

Comment:
Reference the specific product component that this class controls. Also rename to match product component (EntityInsertPanel.tsx).
```suggestion
/**
 * This class automates the UI component defined in <a href="https://github.com/LabKey/labkey-ui-components/blob/master/packages/components/src/components/entities/EntityInsertPanel.tsx">components/entities/EntityInsertPanel.tsx</a>
 * This is the same component (collection of atomic elements) used in insertAssay and createSamples.
 */
public class EntityInsertPanel extends WebDriverComponent<EntityInsertPanel.ElementCache>
```